bin/exec-env

```diff
@@ -1,10 +1,9 @@
 #!/usr/bin/env bash
 
-source $(dirname $(dirname $(dirname $0)))/.asdf/lib/utils.sh
-
 plugin_name="lua"
 
-full_version=$(get_preset_version_for $plugin_name)
+install_path="$(asdf where lua)"
+full_version="$( echo $install_path | rev | cut -d/ -f1)"
```

Comment:
This version is backwards. I get `1.3.5` instead of `3.5.1`.

Reply:
> This version is backwards. I get `1.3.5` instead of `3.5.1`.

I suppose you meant `1.5.3` instead of `1.3.5`?. Anyway will have another go at it shortly. Thanks.

Reply:
My bad, meant to say I got `1.5.3`.